lightning/src/ln/features.rs

```diff
@@ -285,7 +303,8 @@ mod sealed {
 
 	#[cfg(test)]
 	define_feature!(23, UnknownFeature, [TestingContext],
-		"Feature flags for an unknown feature used in testing.");
+		"Feature flags for an unknown feature used in testing.", set_supports_unknown_feature,
+		set_requires_unknown_feature);
```

Comment:
Note that elsewhere we already use `set_required_` and `set_optional_` (though enabled only in testing). I'd prefer to standardize on that naming and remove the one-off definitions for this feature.

Reply:
Hm wait, is this fixed? Or did you want something special for this feature?

Reply:
Ah, never mind. I had thought these may be duplicating `set_required_unknown_bits` and `set_optional_unknown_bits`, but those apply to all contexts not just `TestingContext`.